### ISSUE TITLE
Estimators with industry-specific linear trends

### ISSUE THREAD

**Issue by vinzptrs** [chaisemartinPackages/yatchew_test#16]
Good day,

I am currently trying to apply the Yatchew (1997), de Chaisemartin and D'Haultfoeuille (2024) linearity test to a setting where including industry-specific time trends seems warranted, as is done in the application of the test to Pierce and Schott (2016) in the accompanying dC/DH paper ("Two-way Fixed Effects and Differences-in-Differences Estimators in Heterogeneous Adoption Designs"), section 5.1. 

However, I am not quite sure how to do this in Stata. If I get it right, the dependent variable Y is transformed in a way to reflect a setting with industry-specific linear trends, but I find it hard to follow the exact explanation in the paper. Would it be possible to share (some of) the code that goes along with the application in the paper? 

Thanks and best,
Vinzenz

**Comment by chaisemartinPackages** [comment]
Dear Vinzenz,
This is Diego from Clément de Chaisemartin's RA Team.
Thanks for your interest in the yatchew_test package!
The replication package is currenly under the last round of reviews and it will be ready for publication in two weeks at most.
To this end, we would appreciate your patience while we complete the refactoring of the replication code.
We will make sure to notify you when the replication package will be uploaded to the corresponding [repository](https://github.com/chaisemartinPackages/ReplicationPackages).
Best,
Diego

**Comment by vinzptrs** [comment]
Great, thanks!

**Comment by chaisemartinPackages** [comment]
Dear Vinzenz,
You can find the replication package of Pierce and Schott (2016) from de Chaisemartin and D'Haultfoeuille (2024b) in the [ReplicationPackages Repository](https://raw.githubusercontent.com/chaisemartinPackages/ReplicationPackages/main/pierce_schott.do).
I hope it helps!
Best,
Diego

**Comment by chaisemartinPackages** [comment]
Hi Vinzenz,
Unfortunately, we put down those files due to the fact that a part of the data is proprietary. Nonetheless, if you wish to inquire about how the Y is transformed to account for industry specific linear trends, I would suggest checking the description of the Pierce and Schott application on the new version of the [DID textbook](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4487202) by de Chaisemartin & D'Haultfoeuille. 
The description of how to account for industry-specific linear trends is in pages 170/171.
I hope this helps!
Best,
Diego
